src/BiometryService/BiometryService.csproj

```diff
@@ -45,6 +45,13 @@
 		<PackageReference Include="Xamarin.AndroidX.RecyclerView" Version="1.2.1.7" />
 	</ItemGroup>
 
+	<ItemGroup>
+		<None Include="..\..\README.md">
```

Comment:
This is not enough to include the readme in the nuget package. You need to set the `PackageReadmeFile` too.